### DISCUSSION TITLE
Error when attempting to browse beyond the root directory

### DISCUSSION THREAD

**Discussion by SuperPupperDoggo** [iterate-ch/cyberduck#15399] [Q&A]
I am getting `Not Found. Service Error Message. NoSuchBucket.` when attempting to open folders in my bucket. This does not seem to be a server side issue as my application can access it just fine. The bucket is hosted on Linode at the us-east-1 region and this is occurring across all my buckets.

**Comment by dkocher** [comment]
Do you connect with a _Linode Object Storage_ connection profile provided in _Preferences → Profiles_? Please attach the [transcript](https://docs.cyberduck.io/cyberduck/support/#logging-output).

**Comment by twodom** [comment]
I am experiencing the same issue. I chose the "_Amazon S3_" profile because the bucket I'm connecting to an "_Linode Object Storage_" in Los Angeles, CA(us-lax-1.lindeobjects.com), and I didn't find a match in the profiles. Is it possible that the program does not support the new region "_bucketxx.us-lax-1.linodeobjects.com_"?

**Comment by dkocher** [comment]
The documentation and available connection profiles can be found in our documentation [^1]. I have opened iterate-ch/profiles#70 to add profiles for new regions.

[^1]: https://docs.cyberduck.io/protocols/s3/linode/

**Comment by SuperPupperDoggo** [comment]
Oh. I was not aware you had to do that. From what I can tell the profiles are for specifying the way linode’s S3 servers exactly work, I feel like it should either automatically install the profile or prompt the user to do so when connecting to a linode S3 bucket?

**Reply by dkocher** [reply]
That would be a great addition. I have opened #15408 for this proposal.

**Comment by dkocher** [comment]
Also refer to the documentation from [Linode](https://www.linode.com/docs/products/storage/object-storage/guides/cyberduck/).
